### ISSUE TITLE
export 'useHistory' was not found in 'react-router-dom' - hooks stopped working

### ISSUE THREAD

**Issue by HQ92** [ReactTraining/react-router#7163]
## Version

"react-router": "^5.1.2",
"react-router-dom": "^5.1.2",
"@types/react-router": "^5.1.2",
"@types/react-router-dom": "^5.1.2",

## Steps to reproduce

Use of any hooks within a typescript project. This is working my desktop pc, but on my laptop webpack is failing to build and returning typescript errors. I have checked the typescript files and they are exporting the hooks correctly. However react-router-dom does not seem to have any mention of the hooks within its file that is being imported.

## Expected Behavior

React-router-dom to export the hooks including useRouter, useHistory, useParams, useRouteMatch and so on.

## Actual Behavior

cjs/react-router-dom.js only exports:

exports.BrowserRouter = BrowserRouter;
exports.HashRouter = HashRouter;
exports.Link = Link;
exports.NavLink = NavLink;

Any help would be appreciated! It works on my desktop but that was not a fresh npm install so may be using a stale copy of react-router-dom?

**Comment by timdorr** [comment]
That is not the right version of react-router-dom. I would remove your node_modules and install fresh. 

This isn't an issue with React Router.

**Comment by HQ92** [comment]
@timdorr Do I need to go up to version 6? I was using 5.1.2 before so has something changed? Just want to get it working again haha. Really appreciate the help.

**Comment by timdorr** [comment]
No, 6 is still in alpha. Use 5.1.2 for now.

**Comment by HQ92** [comment]
You're right! I went from ^5.1.2 to 5.1.2 which introduced a load of different issues. I checked my dependencies and pulled my repo from scratch and built everything from scratch and finally my code started showing me some errors from another module that had changed its import to a default one silently in a minor version. I've switched all my dependencies to ones without a tilda so I won't have modules that are silently different across machines. I started panicking as my project was working one day and then the next it stopped and complained about react-router.

Thanks again for the help. For future reference, should I have posted this query on the other forum or was this the right place for it?
